DEVELOPER.md

```diff
@@ -8,36 +8,38 @@ Consult the Table of Contents below to navigate these docs.
 
 ## Table of Contents
 
-* [Developer documentation](#developer-documentation)
-  * [Dependencies](#dependencies)
-      * [The Vue CLI tool](#the-vue-cli-tool)
-  * [Libraries and tools](#libraries-and-tools)
-  * [Folder structure](#folder-structure)
-  * [Getting started](#getting-started)
-      * [Setting up Kuma](#setting-up-kuma)
-        * [Disable anonymous reports](#disable-anonymous-reports)
-      * [Running the GUI](#running-the-gui)
-      * [Building the GUI](#building-the-gui)
-  * [Development](#development)
-      * [Creating a new page view](#creating-a-new-page-view)
-        * [1. Create your view component](#1-create-your-view-component)
-        * [2. Create your REST functions](#2-create-your-rest-functions)
-        * [3. Create your route](#3-create-your-route)
-        * [4. Create your sidebar menu link](#4-create-your-sidebar-menu-link)
-      * [Creating mock data](#creating-mock-data)
-        * [Creating and modifying mocks](#creating-and-modifying-mocks)
-      * [VueX store / State management](#vuex-store--state-management)
-      * [Styles](#styles)
-        * [Variables](#variables)
-        * [Scoped SCSS](#scoped-scss)
-        * [State](#state)
-        * [Tailwind](#tailwind)
+- [Developer documentation](#developer-documentation)
+  - [Table of Contents](#table-of-contents)
+  - [Dependencies](#dependencies)
+    - [The Vue CLI tool](#the-vue-cli-tool)
+  - [Libraries and tools](#libraries-and-tools)
+  - [Folder structure](#folder-structure)
+  - [Getting started](#getting-started)
+    - [Setting up Kuma](#setting-up-kuma)
+      - [Disable anonymous reports](#disable-anonymous-reports)
+    - [Running the GUI](#running-the-gui)
+    - [Building the GUI](#building-the-gui)
+  - [Development](#development)
+    - [Creating a new page view](#creating-a-new-page-view)
+      - [1. Create your view component](#1-create-your-view-component)
+      - [2. Create your REST functions](#2-create-your-rest-functions)
+      - [3. Create your route](#3-create-your-route)
+      - [4. Create your sidebar menu link](#4-create-your-sidebar-menu-link)
+    - [Creating mock data](#creating-mock-data)
+      - [Creating and modifying mocks](#creating-and-modifying-mocks)
+    - [VueX store / State management](#vuex-store--state-management)
+    - [Styles](#styles)
+      - [Variables](#variables)
+      - [Scoped SCSS](#scoped-scss)
+      - [State](#state)
+      - [Tailwind](#tailwind)
+      - [Testing](#testing)
```

Comment:
Btw TOCs are not needed in Github it auto generates one: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/